### PR TITLE
add repositories for event cameras

### PR DESCRIPTION
**Pull request by berndpfrommer**
# Please Add This Package to be indexed in the rosdistro.
event_array_msgs
metavision_driver

ROSDISTRO NAME

rolling
humble

# The source is here:

https://github.com/ros-event-camera/event_array_msgs
https://github.com/ros-event-camera/metavision_driver

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: galactic/distribution.yaml [hunk 1]
```diff
@@ -1027,6 +1027,16 @@ repositories:
       url: https://github.com/OUXT-Polaris/embree_vendor.git
       version: master
     status: developed
+  event_array_msgs:
```
Comment (clalancette) [inline galactic/distribution.yaml:1030]: Please remove all changes to Galactic, since it is end-of-life.

**Review by clalancette — DISMISSED** [review]

File: galactic/distribution.yaml (hunk 1 above)
Comment (berndpfrommer) [reply galactic/distribution.yaml]: done

**Review by audrow — APPROVED** [review]

**Comment by berndpfrommer** [comment]
Is there anything I have to do on my side to move this PR forward? Just want to make sure we are not waiting for each other.

**Review by sloretz — COMMENTED** [review]
- [X] At least one of the following must be present
  - [ ] Top level license file: 
  - [x] Per package license files
- [x] License is [OSI-approved](https://opensource.org/licenses):
- [x] License correctly listed in package.xmls
- [x] Public source repo:
- [x] Source repository contains ROS packages
- [ ] Each package meets [REP-144](https://www.ros.org/reps/rep-0144.html) naming conventions
  - "event_array_msgs" seems borderline to me. On its own I wouldn't know what the messages would be for. Maybe `event_camera_msgs` would be a more specific and more descriptive name?

<details><summary>Package name details</summary>

```console
$ find . -name "package.xml" -exec grep --color=auto -e "<name>" "{}" ";"
  <name>event_array_msgs</name>
  <name>metavision_driver</name>
```
</details>

<details><summary>License details</summary>

```console
$ find . -name "package.xml" -exec grep --color=auto -e "<license>" "{}" "+"
./event_array_msgs/package.xml:  <license>Apache-2</license>
./metavision_driver/package.xml:  <license>Apache-2</license>
```
</details>

**Comment by sloretz** [comment]
> Is there anything I have to do on my side to move this PR forward? Just want to make sure we are not waiting for each other.

I have a concern about the name `event_array_msgs` not meeting REP-144's guidelines. 

The other things we're waiting on are @clalancette to check if his feedback was addressed and Rolling to be out of a sync hold.

**Comment by berndpfrommer** [comment]
Agreed. Will change the package name to event_camera_msgs.

**Comment by berndpfrommer** [comment]
event_array_msgs has been changed to event_camera_msgs

**Review by emersonknapp — APPROVED** [review]

**Comment by emersonknapp** [comment]
Unfortunately missed the window in between holds -

Holding for an upcoming [Humble sync](https://discourse.ros.org/t/preparing-for-humble-sync-and-patch-release-2023-07-21/32427)

**Review by audrow — APPROVED** [review]

**Pull request merged**